thoth/package_extract/image.py

```diff
@@ -464,7 +465,27 @@ def construct_rootfs(dir_path: str, rootfs_path: str) -> list:
     return layers
 
 
-def _get_python_interpreters(path: str) -> dict:
+def _get_absolute_link(root_path: str, path: str, iter: int) -> Optional[str]:
+    """Find the absolute link of the given link."""
+    if iter > _MAX_SYMLINKS:
+        _LOGGER.warning(
+            "Maximum symlink traversal. It should be < 50: %s",
+            str(_MAX_SYMLINKS)
```

Comment:
This will always print `"Maximum symlink traversal. It should be < 50: 50"` You can simplify it to `"Maximum symlink traversal reached"`

Reply:
Thanks, done!